fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatsVersion.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.starrocks.sql.optimizer.statistics;
+
+class StatsVersion {
```

Comment:
can this statsVersion be used for column statistics

Reply:
I think it's ok, and I think we'll need to completely rethink the entire implemention of the statistics collect in the future